### DISCUSSION TITLE
Get the provision progress on remoteOrchestrator.ProvisionAsync

### DISCUSSION THREAD

**Discussion by bearyung** [Mimetis/Dotmim.Sync#905] [Q&A] [answered]
Hi, is there any way(s) I can get the progress (like SyncAgent.SynchronizeAsync) when I am performing remoteOrchestrator.ProvisionAsync? 

Refer to the doc, it seems there should have the "progress" parameter input to the ProvisionAsync() with the below screenshot:
![image](https://user-images.githubusercontent.com/13907070/207534760-edb0eb12-00cd-43a1-a917-0d7a0d274b9a.png)

But I've found that when I trying to do the same thing on my code, VS reported there's no parameter with the name "progress" with the below screenshot:
![image](https://user-images.githubusercontent.com/13907070/207534966-b663104b-9f51-4495-ad27-533a507ab2b9.png)

So I would like to know if there's any ways to get the progress? Thank you.

Michael Yung

**Comment by Mimetis** [comment, marked as answer]
Hello @bearyung 

Indeed, I've disabled the `IProgress<T>` in the last release, as I faced some weird behaviors / issues with it.

I've enabled it again in the last commit, available directly in the source code.
Can you try the last version of the code and see if it fits?

That's being said, it will be part of the next release (`0.9.8`)
